rpc/src/rpc_subscriptions.rs

```diff
@@ -640,6 +656,9 @@ impl RpcSubscriptions {
                             .build()
                             .unwrap();
                         pool.install(|| {
+                            if let Some(rpc_notifier_ready) = rpc_notifier_ready {
+                                rpc_notifier_ready.fetch_or(true, Ordering::Relaxed);
```

Comment:
fn fetch_or(&self, val: bool, order: Ordering) -> bool
Logical “or” with a boolean value.

Performs a logical “or” operation on the current value and the argument val, and sets the new value to the result.

Returns the previous value.